set.mm

```diff
@@ -495674,6 +496088,129 @@ such that every prime ideal contains a prime element (this is a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Algebraic numbers and Minimal polynomials
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c AlgNb minPoly $.
+
+  $( Extend class notation with the algebraic number builder function. $)
+  calgnb $a class AlgNb $.
+
+  $( Extend class notation with the minimal polynomial builder function. $)
+  cminply $a class minPoly $.
+
+  ${
+    $d e f p $.
+    $( Define the minimal polynomial builder function.  This definition is
```

Comment:
Not "minimal polynomial" but "algebraic numbers", perhaps?

Reply:
Of course!